common/src/main/java/com/wynntils/commands/WynntilsCommand.java

```diff
@@ -27,10 +28,14 @@
 public class WynntilsCommand extends CommandBase {
     @Override
     public void register(CommandDispatcher<CommandSourceStack> dispatcher) {
+        throw new UnsupportedOperationException("WynntilsCommand need special treatment");
```

Comment:
Uh oh

Reply:
We need to have a list of all other commands, since we want to create aliases, like `/wynntils token` for `/token`.

Personally I am not sure I like this, but that is how we currently do it.

Reply:
So this was just working out of pure luck before, since we sorted `WynntilsCommand` last...

Reply:
Also, I think we could have like `/wynntils wynntils wynntils wynntils token` before as well, but I have not checked.

Reply:
> Also, I think we could have like `/wynntils wynntils wynntils wynntils token` before as well, but I have not checked.

I think that was handled.

Reply:
Okay, so what do we need to fix this?

Reply:
(I want to keep the option to prefix with /wynntils)

Reply:
> So this was just working out of pure luck before, since we sorted `WynntilsCommand` last...

It was intentional, just not noted. Can't we still do this?

Reply:
I have not removed the functionality to do `/wynntils token`. But to get this to work I send the set of previously registered commands as an explicit argument to the register method in WynntilsCommand, so it does not try to go and query Managers.Command, which is not yet set up.

I could have changed the signature for all commands to get the set of hitherto registered commands, but I thought that was just ... odd.

So everything will work just fine, and we make it explicit that `/wynntils` is a bit special since it will proxy to all other commands. I think this is just fine, and a bit less magic than previously.